db/fixtures/miq_event_definitions.csv

```diff
@@ -209,3 +209,10 @@ containerreplicator_successfulcreate,Replicator Successfully Created Pod,Default
 containerreplicator_compliance_check,Replicator Compliance Check,Default,compliance
 containerreplicator_compliance_passed,Replicator Compliance Passed,Default,compliance
 containerreplicator_compliance_failed,Replicator Compliance Failed,Default,compliance
+
+#
+# Physical Servers Operations
+#
+phs_shutdown,Physical Server Shutdown,Default,phs_operations
+phs_start,Physical Server Start,Default,phs_operations
+phs_reset,Physical Server Reset,Default,phs_operations
```

Comment:
I agree with you @gmcculloug, I use phs because of the suggestions. But this acronym may confuse new contributors for not being widespread or used.